README.md

```diff
@@ -3648,20 +3648,24 @@ resource cleanup when possible.
 <sup>[[link](#percent-s)]</sup>
 
 * <a name="percent-literal-braces"></a>
-  Prefer `()` as delimiters for all `%` literals, except `%r`. Since parentheses
-  often appear inside regular expressions in many scenarios a less common
-  character like `{` might be a better choice for a delimiter, depending on the
+  Prefer `()` as delimiters for string `%` literals, `[]` for array literals and `{}` for `%r`.
```

Comment:
This has become too complex. I'd split it into several rules. One for arrays, one for regular expressions and one for everything else. I'd also add more argumentation - e.g. `[]` is preferable because it's aligned with the standard array literals and so on.